test/Libraries/RevitNodesTests/Elements/ElementTests.cs

```diff
@@ -435,6 +435,38 @@ private static void AssertElementsAreJoined(Element element, Element otherElemen
             Assert.AreEqual(expected, arejoined);
         }
 
+        [Test]
+        [TestModel(@".\Element\elementJoin.rvt")]
+        public void CanSuccessfullySwitchOrderOfTwoJoinedElements()
+        {
+            int cuttingElementId = 208422;
+            int cuttedElementId = 208572;
```

Comment:
grammar issue : `cutted` 
```suggestion
            int cutElementId = 208572;
```